src/jobs/keys-update/keys-update.service.ts

```diff
@@ -142,11 +137,13 @@ export class KeysUpdateService {
         for (const module of modules) {
           const moduleInstance = this.stakingRouterService.getStakingRouterModuleImpl(module.type);
 
-          // At the moment lets think that for all modules it is possible to make decision base on nonce value
+          // At the moment, let's assume that it is possible to make decisions for all modules based on the nonce value
```

Comment:
`nonce` should change in all modules with key changes:
https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/interfaces/IStakingModule.sol#L50-L61

so it's correct to update the keys when the nonce changes